templates/block/layout-builder/block--inline-block--video.html.twig

```diff
@@ -1,13 +1,13 @@
 {% extends "@atomic/block/layout-builder/_layout-builder-block-template.twig" %}
 
+{% set video__width = "site" %}
+{% set video__alignment = content.field_style_alignment.0['#markup']|default('left') %}
+
 {% block content %}
 
   {% if parentNode == 'post' or parentNode == 'event' %}
     {% set video__width = "content" %}
     {% set video__alignment = "center" %}
```

Comment:
I wanted to point out that the code is hard-coding center in posts and events already. I chose to not modify this, so even though left could be selected, in the case of a post or event, it will center. If this should be changed, let me know.